Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -860,6 +860,110 @@ DECLARE_SOA_TABLE(Decays3Body, "AOD", "DECAY3BODY", //! Run 2 cascade table
 using Decays3Body = Decays3Body; //! this defines the current default version
 using Decay3Body = Decays3Body::iterator;
 
+namespace strangenesstracking
+{
+DECLARE_SOA_INDEX_COLUMN_FULL(ITSIdx, Idx, int, Tracks, "");                    //! ITS index
+DECLARE_SOA_INDEX_COLUMN(Cascade, cascadeIdx);                                  //! Cascade index
+DECLARE_SOA_INDEX_COLUMN(V0, v0Idx);                                            //! V0 index
```

Comment:
```suggestion
DECLARE_SOA_INDEX_COLUMN(Cascade, cascade);                                  //! Cascade index
DECLARE_SOA_INDEX_COLUMN(V0, v0);                                            //! V0 index
```
Nowhere else we use `idx` in the column names